### ISSUE TITLE
Profile Screen Nametag

### ISSUE THREAD

**Issue by ajsarkar28** [department-of-veterans-affairs/va-mobile-app#8092]
AC:
- Update the Veteran Status Card in Profile to reflect the designs in this ticket

- [Link to mockup](https://www.figma.com/file/q4oTyzyLfYrZVz4CACzVLA/Personalization-Visual-Design---Working---VA-Mobile?type=design&node-id=3732-2456&mode=design&t=U06zmdmHq3p9asLg-4)
- Veteran name should be announced as a heading by screen reader
- Vet Status component: Make sure the icon is a fixed size; does not resize on display zoom.

<img width="378" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNkdnQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4b481a716bdf0122e633016882029ce7662e1554/image.png" alt="image.png" />

**Comment by theodur** [comment]
@wavelaurenrussell Did you want the icon in the Nametag fixed also? Like how you mentioned [here](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7599#issuecomment-1997657960)?

**Comment by wavelaurenrussell** [comment]
Yes! I will ensure Figma reflects that too.

**Comment by TKDickson** [comment]
@dumathane the veteran's name should be announced as a heading by screenreader; this looks clean otherwise.

**Comment by TKDickson** [comment]
Alright I fixed the heading announcement (yay!) but we're not handling not-getting-name back from the user endpoint as well as we could: holding space for the name (regardless of SR on or not), and also announcing "heading" on the blank space:

<img width="372" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOC9pQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--62ee498d95a47d0e2b6231bcb33f4677e49d9505/image.png" alt="image.png" />

**Comment by TKDickson** [comment]
Fixed, thanks @theodur . This is good to merge to XL (and includes also-good-to-merge tickets #7877 and #7852)

**Comment by TKDickson** [comment]
Merged to XL, closing
